### PR TITLE
view margin-top 3.5% -> 50px so it's not dependant on screen width

### LINKED ISSUE #1: Correct the header of the template (HTML)
On Higher resolution screens, there is gap between the header and main content.
image

### PR DESCRIPTION
**Pull request by xvvvyz**
I believe this fixes #1.

### CODE REVIEW

File: public/css/style.css [hunk 1]
```diff
@@ -32,7 +32,7 @@ body a:hover{
 }
 /*--view--*/
 .view{
-    margin-top:3.5%;
+    margin-top: 50px;
```
Comment (manishrw) [inline public/css/style.css:35]: this won't be applicable to all the screen sizes. Did you test on all the screen sizes/ resolutions ?

File: public/css/style.css [hunk 2]
```diff
@@ -12,7 +12,7 @@ body a{
 	text-decoration:none;
 }
 h1,h2,h3,h4,h5,h6{
-	margin:0;
```
Comment (manishrw) [inline public/css/style.css:15]: Why is there a change here ?

File: public/css/style.css [hunk 3]
```diff
@@ -291,7 +291,7 @@ p {
     margin-bottom: 2.2em;
     background-color: #2F2C06;
 }
-.progress-bar { 
+.progress-bar {
```
Comment (manishrw) [inline public/css/style.css:294]: Similarly here, extra spaces have been added. Could you remove ?

File: public/css/style.css (hunk 1 above)
Comment (xvvvyz) [reply public/css/style.css:35]: Seems to work okay.
screen shot 2016-10-26 at 11 50 00

File: public/css/style.css (hunk 2 above)
Comment (xvvvyz) [reply public/css/style.css:15]: Removed extra whitespace at the end of the line.

File: public/css/style.css (hunk 3 above)
Comment (xvvvyz) [reply public/css/style.css:294]: No, extra space has been removed.

**Comment by xvvvyz** [comment]
All of the other changes are just removing extra whitespace at the end of a bunch of lines. My editor does that automatically.

**Pull request merged**